lightning_examples/finetuning-scheduler/finetuning-scheduler.py

```diff
@@ -0,0 +1,580 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.13.1
+#   kernelspec:
+#     display_name: 'Python 3.7.11 64-bit (''pldev_tutorials'': conda)'
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# ## Scheduled Finetuning
+#
```

Comment:
```suggestion
```

Reply:
thought this top section might need to be removed, have done so now, thanks!